Dockerfile

```diff
@@ -62,8 +58,21 @@ RUN apt-get update \
     libunwind8 \
     libuuid1 \
     zlib1g \
+    xz-utils \
   && rm -rf /var/lib/apt/lists/*
 
+# Download and unpack Node.js v10
+RUN mkdir -p /opt \
+  && curl -L https://nodejs.org/dist/v10.16.0/node-v10.16.0-linux-x64.tar.xz -o /opt/node-v10.16.0-linux-x64.tar.xz \
+  && tar -C /opt -xJf /opt/node-v10.16.0-linux-x64.tar.xz \
+  && rm -f /opt/node-v10.16.0-linux-x64.tar.xz
+ENV PATH /opt/node-v10.16.0-linux-x64/bin:$PATH
+
+# Install google-gax for Node.js
+RUN npm install -g google-gax@^1.2.1
```

Comment:
Oh bother.  What's going to keep that version up to date?

Reply:
Well... semver `^1.2.1` will allow everything up to `2.0`, and it actually prevents me from possible breaking changes in `google-gax`. It's pretty much the same way as we do in `package.json`, isn't it?